main.nf

```diff
@@ -729,20 +754,14 @@ if( !params.classifier ){
 			.set { ch_ref_database }		
 	}
 
-	process make_classifier {
-		publishDir "${params.outdir}/DB/", mode: params.publish_dir_mode, 
-		saveAs: {filename -> 
-			if (filename.indexOf("${params.FW_primer}-${params.RV_primer}-${params.dereplication}-classifier.qza") == 0) filename
-			else if(params.keepIntermediates) filename 
-			else null}
+	process clfr_extract_seq {
```

Comment:
just for ease of reading
```suggestion
	process classifier_extract_seq {
```
if you accept this, please replace all occurrences of `clfr` to `classifier`, e.g. in `clfr_train`, in main.nf, configs etc.